CHANGELOG.md

```diff
@@ -15,11 +15,14 @@ Misc:
 Features:
 
 - [#2361](https://github.com/rails-api/active_model_serializers/pull/2361) Add `ActiveModelSerializers.config.use_sha1_digests` to allow customization of the hashing algorithm used for serializer caching (@alexzherdev)
+- [#2398](https://github.com/rails-api/active_model_serializers/pull/2398) - rails 6.1 (@ritikesh)
```

Comment:
@ritikesh it's considered a bad practice to edit the CHANGELOG yourself unless a maintainer asks you, fwiw

Reply:
Cleaned up the mess. My bad, should have confirmed. 👍